ProductService.js

```diff
@@ -0,0 +1,136 @@
+async function getProductList(page, pageSize, keyword) {
+  let data;
```

Comment:
```suggestion
async function getProductList(page, pageSize, keyword) {
  let data;
  const API_URL = "https://panda-market-api-crud.vercel.app"
```
api를 콜할 때 사용하는 api주소가 반복적으로 사용되고 있습니다. 이럴 때는 상수로서 선언하고 해당 상수를 필요한 곳에서 가져다 사용하면 좋을 것 같습니다.
실제로 이런 URL 값들은 환경변수에 저장하고 사용하는 경우가 많습니다.
나머지 api에 대해서 상수로 대체해서 한 번 변경해 보세요.